test/check-storage-home-reuse

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/python3
+#
+# Copyright (C) 2024 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; If not, see <http://www.gnu.org/licenses/>.
+
+import os
+
+from anacondalib import VirtInstallMachineCase
+from installer import Installer
+from review import Review
+from storage import Storage
+from storagelib import StorageCase  # pylint: disable=import-error
+from testlib import nondestructive, skipImage, test_main  # pylint: disable=import-error
+
+TEST_DIR = os.path.dirname(__file__)
+ROOT_DIR = os.path.dirname(TEST_DIR)
+BOTS_DIR = f'{ROOT_DIR}/bots'
+
+
+@nondestructive
+class TestHomeReuseFedora(VirtInstallMachineCase, StorageCase):
+    disk_image = "fedora-rawhide"
+    efi = False
+
+    def testBasic(self):
+        b = self.browser
+        m = self.machine
+        i = Installer(b, m, scenario="home-reuse")
+        s = Storage(b, m)
+        r = Review(b, m)
+
+        dev="vda"
+
+        i.open()
+        i.reach(i.steps.INSTALLATION_METHOD)
+        s.rescan_disks()
+
+        #s.wait_scenario_available("home-reuse", False)
+
+        s.set_partitioning("home-reuse")
+        i.reach(i.steps.REVIEW)
+
+
+        # check selected disks are shown
+        r.check_disk(dev, f"16.1 GB {dev} (0x1af4)")
+        r.check_disk_row(dev, parent=f"{dev}3", action="delete")
+        r.check_disk_row(dev, parent=f"{dev}1", action="delete")
+        r.check_disk_row(dev, "/boot", f"{dev}3", "1.07 GB", True, "xfs", is_encrypted=False)
+        r.check_disk_row(dev, "/", f"{dev}4", "12.8 GB", True, "btrfs", is_encrypted=False)
+        r.check_disk_row(dev, "/home", f"{dev}4", "12.8 GB", False, "btrfs", is_encrypted=False,
+                         action="reuse")
+
+        # TODO pixel test
+
+    @skipImage("btrfs support missing on fedora-eln image", "fedora-eln-boot")
```

Comment:
We dont really test on ELN anymore. You can just ignore it. We can also drop the rest of the decorator around ELN.

Reply:
I'll remove it in a separate PR.